sql/item_mods.sql

```diff
@@ -34401,6 +34436,23 @@ INSERT INTO `item_mods` VALUES (25823,31,48);     -- Magic Evasion+48
 INSERT INTO `item_mods` VALUES (25823,68,19);     -- Evasion+19
 INSERT INTO `item_mods` VALUES (25823,311,49);    -- Magic Damage+49
 INSERT INTO `item_mods` VALUES (25823,384,400);    -- Haste +40
+INSERT INTO `item_mods` VALUES (25842,1,114);     -- Herculean Trousers: DEF+114
+INSERT INTO `item_mods` VALUES (25842,2,38);      -- HP+38
+INSERT INTO `item_mods` VALUES (25842,8,33);      -- STR+33
+INSERT INTO `item_mods` VALUES (25842,10,16);     -- VIT+16
+INSERT INTO `item_mods` VALUES (25842,11,32);     -- AGI+32
+INSERT INTO `item_mods` VALUES (25842,12,29);     -- INT+29
+INSERT INTO `item_mods` VALUES (25842,13,15);     -- MND+15
+INSERT INTO `item_mods` VALUES (25842,14,10);     -- CHR+10
+INSERT INTO `item_mods` VALUES (25842,23,15);     -- Attack+15
+INSERT INTO `item_mods` VALUES (25842,24,15);     -- Ranged Attack+15
+INSERT INTO `item_mods` VALUES (25842,68,62);     -- Evasion+62
+INSERT INTO `item_mods` VALUES (25842,31,75);     -- Magic Evasion+75
+INSERT INTO `item_mods` VALUES (25842,29,5);      -- Magic Def. Bonus+5
+INSERT INTO `item_mods` VALUES (25842,384,600);   -- Haste+6%
+INSERT INTO `item_mods` VALUES (25842,73,4);      -- Store TP+4
+INSERT INTO `item_mods` VALUES (25842,27,-4);     -- Enmity-4
+INSERT INTO `item_mods` VALUES (25860,161,-2);    -- Physical Damage Taken-2%
```

Comment:
Looks like wrong ID slipped in here.

Reply:
oops uploaded the same file twice... the second was supposed to be that fix -_-